pyron-plugin-impl/src/test/resources/plugins/transformer/rules.json

```diff
@@ -133,6 +133,131 @@
               }
             }
           ]
+        },
+        {
+          "method": "GET",
+          "pathPattern": "/dyn-header-can-find-multiple-params-and-reorder-them-in-output",
+          "requestPlugins": [
+            {
+              "name": "transform-request",
+              "conf": {
+                "body": {},
```

Comment:
I think you can drop the body here

Reply:
I think it must be there. During config read, it will otherwise see no "body" entry while "$body" is referenced. We only set up contents of the body in the test, but it must "exist" during config init.

Reply:
ok